src/pages/Users/UserLogin/index.jsx

```diff
@@ -135,17 +182,49 @@ const UserLogin = () => {
               <div className='sign-up text-center mb-3'>
                 <p>Sign Up</p>
               </div>
-              <MDBInput wrapperClass='mb-4' placeholder='Name' id='form2' type='name'/>
-              <MDBInput wrapperClass='mb-4' placeholder='Username' id='form2' type='username'/>
-              <MDBInput wrapperClass='mb-4' placeholder='Email' id='form2' type='email'/>
-              <MDBInput wrapperClass='mb-4' placeholder='Password' id='form2' type='password'/>
-              <MDBInput wrapperClass='mb-4' placeholder='Repeat Password' id='form2' type='password'/>
 
+              <form>
```

Comment:
Extract the login and signup part of the forms to separate components to clean up this file. Its too busy as it is right now.